packages/wagmi/future/hooks/pools/actions/getAllPoolsCodeMap.ts

```diff
@@ -30,7 +30,10 @@ export const getAllPoolsCodeMap = async ({ currencyA, currencyB, chainId }: Omit
     liquidityProviders.push(LiquidityProviders.SushiSwapV3)
   }
   dataFetcher.startDataFetching(liquidityProviders)
-  await dataFetcher.fetchPoolsForToken(currencyA, currencyB)
+  if (currencyA.chainId === currencyB.chainId) {
```

Comment:
This check should probably be an invariant inside the DataFetcher itself

Reply:
Yeah I kept seeing invariant failed CHAINS